.github/workflows/release-develop.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           node scripts/syncLocalDependencies.js $version
           echo "Syncing yarn workspace"
           yarn
-      - run: yarn build --configuration=production
+      - run: yarn build
```

Comment:
This is not required anymore and it can be confusing. That was added for @nx/esbuild but not used anymore